### DISCUSSION TITLE
ComfyUI_windows_portable 0.2.7 @win11 The interface is like crashing (in google chrome)

### DISCUSSION THREAD

**Discussion by coddz** [comfyanonymous/ComfyUI#5545] [General]
The mouse cannot drag nodes, and the mouse wheel cannot zoom in or out
![Snipaste_2024-11-09_02-21-46](https://github.com/user-attachments/assets/1431ce04-8eec-4249-bf8d-5631963eed2b)

**Comment by coddz** [comment]
There is also an old version of Comfyui on this machine, which is operating normally.

Env: 4090 Windows 11
new Comfyui: 0.2.7 torch2.5.1 cu12.4
old Comfyui:  torch2.3.1 cu12.1

**Comment by LukeG89** [comment]
Have you updated ComfyUI and all custom nodes?
If so, have you tried disabling all your nodes with `--disable-all-custom-nodes` argument?
There could be a custom node that is conflicting.

**Reply by coddz** [reply]
> Have you updated ComfyUI and all custom nodes? If so, have you tried disabling all your nodes with `--disable-all-custom-nodes` argument? There could be a custom node that is conflicting.

@LukeG89 Thank you LukeG89 very much. I only copy two custom node: Comfyui-manager and language translater(AIGODLIKE-ComfyUI-Translation) and The interface is like crashing as usual after disabling the two node.

I tried Comfyui 0.2.2 (python 3.11+torch 2.4.1 cu12.4), The interface is like crashing too. I only unpacked the Comfyui 0.2.2, renew the comfyui by update_comfyui.bat , didn't have any custom node.

So I think another reason about the wrong, the error is in comfyui-frontend?

**Reply by LukeG89** [reply]
> So I think another reason about the wrong, the error is in comfyui-frontend?

Yeah, this is probably related to frontend. Open a bug report here: https://github.com/Comfy-Org/ComfyUI_frontend/issues

**Reply by ltdrdata** [reply]
Some custom nodes operate by directly copying front scripts to web/extensions, so disabling them doesn't work.
Please remove all directories in web/extensions except for core.

**Comment by coddz** [comment]
the pure Comfyui 0.2.2 windows_portable runs well:
reinstalled Comfyui 0.2.2 (python 3.11+torch 2.4.1 cu12.4) and do not renew the comfyui by update_comfyui.bat, didn't have any custom node.

can I renew the comfyui by update_comfyui.bat?

**Reply by coddz** [reply]
![Snipaste_2024-11-09_13-04-55](https://github.com/user-attachments/assets/2dd8a572-8924-428e-8291-d545c9ab818a)

I suspect the problem lies with comfyui front? But my other comfyui (python 3.10 torch 2.3.1+cu12.1) still runs well after upgrading to the latest patch.

**Comment by ltdrdata** [comment]
Since this is an issue caused by custom nodes, I'm closing this.
